README.md

```diff
@@ -118,3 +118,26 @@ Additionally, some globals have been added to the testing environment to streaml
 * `expect` - Corresponds with `require('chai').expect`. [Info](http://chaijs.com/api/bdd/)
 * `faker` - Corresponds with `require('faker')`. [Info](https://github.com/marak/Faker.js/)
 * `sandbox` - Corresponds with `require('sinon').sandbox`. Should be used anytime when wanting to create a `sinon.spy` or `sinon.stub` as it can be easily used to reset/restore multiple spys and stubs. [Info](http://sinonjs.org/releases/v4.1.6/sandbox/)
+
+## Contributing/Publishing
+
+There are certain steps that should be taken when contributing and publishing a new release of the `contxt-sdk`.
+
+### Contributing
+
+Be sure to run `npm run build` before putting your pull request up for code review. While this is command is run in
+preparation of publishing to NPM, it autogenerates the documentation and should be committed into GitHub as well.
```

Comment:
Maybe change this line to be `autogenerates the documentation, which should be committed to GitHub as well`? I think maybe we could get rid of `to GitHub` too since thats just where the remote is hosted, but that's kind of splitting technical hairs 😂

Reply:
Same comment as above, I also dropped `GitHub` and just used `source control`